entrypoint.sh

```diff
@@ -21,9 +21,30 @@ if [ "$AVD" ]; then
     socat tcp-listen:5555,bind=127.0.0.1,fork tcp:$AVD & SOCAT_PID=$!
 fi
 
-start
+url="$JENKINS_MASTER_URL/swarm/swarm-client.jar"
 
-if [ "$AVD_PID" ]; then
-    kill $SOCAT_PID
-fi
+startTime=$(date +%s)
+waitTime=600
```

Comment:
if it is in seconds then 60 should be enough. I'd recommend to move it onto the env var with 60 as default value